### ISSUE TITLE
Add additional Columns to search by, Sort the Returned List by column/columns set by designer

### ISSUE THREAD

**Issue by Cold-Seven** [markcunninghamuk/PCF_Library#7]
Would it be possible to add more than one column for the search field to be searched against? 

And likewise, would it be possible to add a sort option to the selected column/columns?

Great control!!

**Comment by markcunninghamuk** [comment]
yes I believe I can do it, i would need to think about the design

**Comment by markcunninghamuk** [comment]
I would love to enhance this, I will look at in in the next few months.

**Comment by Krodgersminto** [comment]
@markcunninghamuk I would also be interested in a sort as currently if i add multiple records in a certain order and save and re-open the order is changed. For example if i add values; 10,5,7,3,1 save and re-open the display order changes it as 1,3,5,7,10. Being able to keep the original order would be great.

Thank you
